maps/contributors.coffee

```diff
@@ -0,0 +1,8 @@
+module.exports =
+  name: 'Jenna Sutela'
+    location: 'Berlin'
+    links: ['(Website)[http://jennasutela.com/]', '(twitter)[https://twitter.com/jennaemilia]']
+    bio: "Jenna Sutela is an artist and writer based between Helsinki and Berlin. Her projects seek to identify and react to precarious social and material moments—most recently, the relationship between the body and its technologically-mediated environment. She is a contributor to several international publications, such as _Frieze_ and _Harvard Design Magazine_, and her work has been presented at, for example, _Haus der Kulturen der Welt_, _Museum of Contemporary Art Kiasma_ and _South London Gallery_."
+    azone_uersname: null
+    azone_reviews: ['(Jenna Sutela Launches Roko’s Basilisk Whisperer into the Åzone Futures Market)[http://azone.guggenheim.org/review/11241]']
+    azone_funds: '(General Intelligence Fund)[http://azone.guggenheim.org/investing/funds/general-intelligence]'
```

Comment:
So this whole thing should probably be an array (since we expect multiple contributors) and there doesn't really need to be any extra indention. So something like:

```
module.exports = [
  name: 'Jenna Sutela'
  location: 'Berlin'
  links: ['(Website)[http://jennasutela.com/]', '(twitter)[https://twitter.com/jennaemilia]']
  bio: "Jenna Sutela is an artist and writer based between Helsinki and Berlin. Her projects seek to identify and react to precarious social and material moments—most recently, the relationship between the body and its technologically-mediated environment. She is a contributor to several international publications, such as _Frieze_ and _Harvard Design Magazine_, and her work has been presented at, for example, _Haus der Kulturen der Welt_, _Museum of Contemporary Art Kiasma_ and _South London Gallery_."
  azone_uersname: null
  azone_reviews: ['(Jenna Sutela Launches Roko’s Basilisk Whisperer into the Åzone Futures Market)[http://azone.guggenheim.org/review/11241]']
  azone_funds: '(General Intelligence Fund)[http://azone.guggenheim.org/investing/funds/general-intelligence]'
  ,
  name: 'Jenna Sutela'
  location: 'Berlin'
  links: ['(Website)[http://jennasutela.com/]', '(twitter)[https://twitter.com/jennaemilia]']
  bio: "Jenna Sutela is an artist and writer based between Helsinki and Berlin. Her projects seek to identify and react to precarious social and material moments—most recently, the relationship between the body and its technologically-mediated environment. She is a contributor to several international publications, such as _Frieze_ and _Harvard Design Magazine_, and her work has been presented at, for example, _Haus der Kulturen der Welt_, _Museum of Contemporary Art Kiasma_ and _South London Gallery_."
  azone_uersname: null
  azone_reviews: ['(Jenna Sutela Launches Roko’s Basilisk Whisperer into the Åzone Futures Market)[http://azone.guggenheim.org/review/11241]']
  azone_funds: '(General Intelligence Fund)[http://azone.guggenheim.org/investing/funds/general-intelligence]'
]
```